### PR TITLE
fix: networkId is now taken from the network magic

### PR DESCRIPTION
**Pull request by AlanVerbner**
# Description

Closes: #238

# Proposed Solution

As stated in #238 

# Important Changes Introduced

n/a

# Testing

n/a

### CODE REVIEW

**Review by rhyslbw — CHANGES_REQUESTED** [review]
After thinking about this we should also map `testnet`, which covers the two primary public networks, and matches what we offer to build out of the box. This means it would not be a breaking change.

Also, the reverse mapping needs to be done to allow the two special cases to be provided in requests under `network_identifier.network`

**Comment by AlanVerbner** [comment]
@rhyslbw can you update #238 so I can change this one please?

File: cardano-rosetta-server/src/server/services/network-service.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Logger } from 'fastify';
 import { Block, GenesisBlock, Network } from '../models';
+import { MAIN_TESTNET_NETWORK_MAGIC } from '../utils/constants';
```
Comment (rhyslbw) [inline cardano-rosetta-server/src/server/services/network-service.ts:3]:
```suggestion
import { TESTNET_NETWORK_MAGIC } from '../utils/constants';
```

File: cardano-rosetta-server/src/server/utils/constants.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export const SUCCESS_STATUS = 'success';
 
 export const MAINNET = 'mainnet';
 export const TESTNET = 'testnet';
+export const MAIN_TESTNET_NETWORK_MAGIC = 1097911063;
```
Comment (rhyslbw) [inline cardano-rosetta-server/src/server/utils/constants.ts:27]: Mainnet network magic is `764824073`

Comment (rhyslbw) [inline cardano-rosetta-server/src/server/utils/constants.ts:27]:
```suggestion
export const TESTNET_NETWORK_MAGIC = 1097911063;
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Let's just use the network name 'testnet', not 'main testnet'
All networks have magic, so just treat mainnet and testnet equally. The idea here is that you can send a request to any Cardano Rosetta instance using the magic, and still get the same result as sending the human friendly name.

**Comment by rhyslbw** [comment]
Ignore the change request, this works

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**